py3status/modules/weather_owm.py

```diff
@@ -31,7 +31,7 @@
         (default 600)
     forecast_days: Number of days to include in the forecast, including today
         (regardless of the 'forecast_include_today' flag)
-        (default 0)
+        (default 3)
     forecast_include_today: Include today in the forecast? (Boolean)
```

Comment:
I agree with @tobes, the end user may want to have the current weather in both.

Also, this option is for better cross compatibility with `weather_yahoo`, since it also has this option.